RELEASENOTES.md

```diff
@@ -4,6 +4,41 @@ This file contains highlights and announcements covering all components.
 For more details see `CHANGELOG.md` files maintained in the root source
 directory of each individual package.
 
+## 1.10.0
+
+* Bumped the package versions of `System.Diagnostic.DiagnosticSource` and other
+  Microsoft.Extensions.* packages to `9.0.0`.
+
+* Added support for new APIs introduced in `System.Diagnostics.DiagnosticSource`
+  `9.0.0`:
+
+  * [InstrumentAdvice&lt;T&gt;](https://learn.microsoft.com/dotnet/api/system.diagnostics.metrics.instrumentadvice-1)
```

Comment:
Are you planning to provide our documentation link for InstrumentAdvice once it is ready?

Reply:
No I'm planning to have you do it! j/k We can update this once it lands. I'm doing this now because I want to test README+RELEASENOTES+CHANGELOG content being included on NuGet when we push 1.10.0-rc.1 packages.